hw3/Task_13.ts

```diff
@@ -0,0 +1,31 @@
+//Task 13
+// Напишем программу для туристического терминала.
+// Создать переменную для стороны света в которую пользовател хотел отправиться.
+// После ввода данных (присвоение переменной определенного значения) мы должны вывести в консоль сообщение из списка.
+// Если пользователь ввел неверные данные, выведите сообщение в консоль чтобы он попробовал еще раз.
+// + `юг` на юг пойдешь счастье найдешь
+// + `север` на север пойдешь много денег найдешь
+// + `запад` на запад пойдешь верного друга найдешь
+// + `восток` на восток пойдешь разработчиком станешь
+//
+// Используйте конструкцию switch
+
+let goTo: string = 'восток';
+
+switch (goTo) {
+    case 'юг':
+        console.log('На юг пойдешь счастье найдешь')
+        break;
+    case 'север':
+        console.log('На север пойдешь много денег найдешь')
+        break;
+    case 'запад':
+        console.log('На запад пойдешь верного друга найдешь')
+        break;
+    case 'восток':
+        console.log('На восток пойдешь разработчиком станешь')
+        break;
+    default:
+        console.log('Попробуй еще раз');
+        break;
```

Comment:
I guess no need to break in default block

Reply:
Agree, fixed.